bentoml/_internal/utils/analytics/schemas.py

```diff
@@ -68,6 +75,11 @@ def get_client_info(
         return new_client_info
 
 
+@lru_cache(maxsize=1)
```

Comment:
`maxsize=1` is useless here because function has a parameter, only repeated calls with same parameter can hit the cache

Reply:
i forgot to remove it